### DISCUSSION TITLE
请教一下关于rime lua的问题

### DISCUSSION THREAD

**Discussion by tmbmyeue** [amorphobia/openfly#51] [一般讨论/General]
个人出于兴趣开始上手rime，开始后才发现rime的文档是真心少啊，尤其关于lua部分的，网上只找到几个例子，对于env, context, config的描述文档根本没有，所以想请教同好，这个项目这么复杂的lua processor 你是参考什么文档编写的呢？恳盼回答，不胜感激！！！

**Comment by amorphobia** [comment]
一开始主要是搜各种例子，然后就是问其他大佬比如 [librime-lua](https://github.com/hchunhui/librime-lua) 的作者 [hchunhui](https://github.com/hchunhui) 以及贡献者 [shewer](https://github.com/shewer)

搜一下 [librime-lua 的 issue](https://github.com/hchunhui/librime-lua/issues?q=is%3Aissue), 有很多问题已经有人提过了

[Rime tg 群](https://t.me/loverime) 也可以去问问

最后就是看 librime-lua 以及 [librime](https://github.com/rime/librime) 源代码，我甚至也发现当时的 librime-lua 不能实现的功能，自己提 pr 了的
